modules/appeals_api/config/routes.rb

```diff
@@ -4,7 +4,13 @@
   match '/v0/*path', to: 'application#cors_preflight', via: [:options]
 
   namespace :v0, defaults: { format: 'json' } do
-    resources :appeals, only: [:index]
+    resources :appeals, only: [:index] do
+      collection do
+        get 'higher_level_reviews/:uuid', to: 'appeals#show_higher_level_review'
+        get 'intake_statuses/:intake_id', to: 'appeals#show_intake_status'
+      end
```

Comment:
I believe it would have been more RESTy to do something similar to

```
    resources :appeals, only: [:index] do
        resources 'higher_level_reviews/:uuid', only: :show
        resources 'intake_statuses/:intake_id',  only: :show
```

but talked to @camerontesterman and they decided against it earlier on. Seems like a big ask to change at this point.